docs/mapview.md

```diff
@@ -52,6 +52,8 @@
 | `isAccessibilityElement`          | `Boolean`                            | `false`      | Determines whether the MapView captures VoiceOver touches or forwards them to children. When `true`, map markers are not visible to VoiceOver. **Note:** iOS Maps only.                                                                                                                                                                                                                                                                                                                                                          |
 | `cameraZoomRange`                 | `CameraZoomRange`                    |              | Map camera distance limits. `minCenterCoordinateDistance` for minimum distance, `maxCenterCoordinateDistance` for maximum, `animated` for animated zoom range changes. Takes precedence if conflicting with `minZoomLevel`, `maxZoomLevel`. **Note**: iOS 13.0+ only.                                                                                                                                                                                                                                                            |
+| `googleMapId`                     | string                               |              | A map ID is a unique identifier that represents a single instance of a Google Map. For more informations check [Google create a map ID documentation](https://developers.google.com/maps/documentation/get-map-id#create-a-map-id) <br /> **Note**: Google Maps only.                                                                                                                                                                                                                                                            |
+| `classicalGoogleMarkers`          | boolean                              | `false`      | If 'true' map will use old MapMarkers instead of AdvancedMapMarkers <br /> **Note**: Android Google Maps only.                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
I think this should be true by default. Consumers of AdvancedMapMarkers need to know that they won't work without specifying a googleMapId first.

There may be another way to document this, but it should be a highly visible:
https://developers.google.com/maps/documentation/android-sdk/advanced-markers/start#check_map_capabilities_required